include/circt/Dialect/SV/SVInOutOps.td

```diff
@@ -23,7 +23,8 @@ def WireOp : SVOp<"wire", [NonProceduralOp,
      See SV Spec 6.7, pp97.
     }];
 
-  let arguments = (ins StrAttr:$name, OptionalAttr<SymbolNameAttr>:$inner_sym);
+  let arguments = (ins StrAttr:$name, OptionalAttr<SymbolNameAttr>:$inner_sym,
+                       OptionalAttr<SVAttributeArrayAttr>:$svAttributes);
```

Comment:
Should this be an `OptionalAttr` or a `DefaultValuedAttr<SVAttributeArrayAttr, "{}">`? Chris would usually push me away from optionality to avoid the degeneracy of `None` vs. `Optional`. However, the code-gen here may be same as I don't think `OptionalAttr` produces an `Option`.

Reply:
Indeed it does not, so the codegen is the same.